### DISCUSSION TITLE
lots of features needed

### DISCUSSION THREAD

**Discussion by SiddharthShyniben** [AnishDe12020/devkit#4] [Ideas]
1. Base64 encoder and decoder
2. Fake JSON generator
3. JSON viewer

**Comment by SiddharthShyniben** [comment]
4. Code to syntax highlighted HTML

**Reply by AnishDe12020** [reply]
I don't think I understand this one well, can you explain it to me?

**Reply by SiddharthShyniben** [reply]
You add code in a textarea and select a language, then you get code in HTML with syntax highlighting. Like https://code.google.com/p/google-code-prettify/

**Reply by AnishDe12020** [reply]
So you put in code in any language and it shows the code prettified and syntax highlighted?

**Reply by SiddharthShyniben** [reply]
Nothing, nvm! Just a misunderstanding by me

**Comment by SiddharthShyniben** [comment]
5. Classic lorem ipsum

**Comment by SiddharthShyniben** [comment]
6. Colour picker in many formats

**Reply by AnishDe12020** [reply]
A color picker sounds good but implementing one that will work OS-wide is hard because this is a web application, do let me know if you are aware of any workarounds

**Reply by SiddharthShyniben** [reply]
What do you mean os-wide? It's a website right?

**Reply by AnishDe12020** [reply]
Yep, it is. Implementing a color picker won't really work because it won't work os-wide because it is a website. A color converter, contrast checker is something I might look into though

**Reply by SiddharthShyniben** [reply]
I guess you didn't get me since I didn't explain correctly 🤦🏼‍♂️ 

I didn't mean a literal eyedrop color picker, I meant a color picker like https://coolors.co/color-picker where we pick from a picker like this:


![Screen Shot 2021-11-04 at 7 46 00 PM](https://user-images.githubusercontent.com/76640268/140329848-753ed827-0ad9-4b19-833c-323d231ae217.png)

We should be able to pick the color like so, and also edit the HSL, RGB, etc.

**Reply by AnishDe12020** [reply]
Sounds good

**Comment by AnishDe12020** [comment]
Wow, that is a great list, I will add it to the roadmap and start working on them one by one
